_api-reference/cluster-settings.md

```diff
@@ -26,30 +45,64 @@ PUT _cluster/settings
+cluster.info.update.interval | Time unit | This sets how often OpenSearch should check disk usage for each node in the cluster. Default is `30s`.
+cluster.routing.allocation.include.<attribute> | Enum | Allocate shards to a node whose `attribute` has at least one of the included comma-separated values.
+cluster.routing.allocation.require.<attribute> | Enum | Only allocate shards to a node whose `attribute` has all of the included comma-separated values.
+cluster.routing.allocation.exclude.<attribute> | Enum | Do not allocate shards to a node whose `attribute` has any of the included comma-separated values. The cluster allocation settings support the following built-in attributes: <br /> <br /> `_name` - Match nodes by node name. <br /> <br /> `_host_ip` - Match nodes by host IP address. <br /> <br /> `_publish_ip` - Match nodes by publish IP address. <br /> <br /> `_ip` - Match either `_host_ip` or `_publish_ip`.   <br /> <br /> `_host` - Match nodes by hostname. <br /> <br /> `_id` - Match nodes by node id. <br /> <br /> `_tier` - Match nodes by data tier role.
+cluster.blocks.read_only | Boolean | Sets the entire cluster to read-only. Default is `false`.
+cluster.blocks.read_only_allow_delete | Boolean | Similar to `cluster.blocks.read_only`, but allows you to delete indexes.
+cluster.max_shards_per_node | Integer | This limits the total number of primary and replica shards for the cluster. The limit is calculated as follows: `cluster.max_shards_per_node` times the number of non-frozen data nodes. Shards for closed indexes do not count toward this limit. Default is `1000`.
+cluster.persistent_tasks.allocation.enable | String | Enables or disables allocation for persistent tasks: <br /> <br /> `all` - Allows persistent tasks to be assigned to nodes. <br /> <br /> `none` - No allocations are allowed for persistent tasks. This does not affect persistent tasks already being run. <br /> <br /> Default is `all`.
+cluster.persistent_tasks.allocation.recheck_interval | Time unit | The cluster manager automatically checks whether or not persistent tasks need to be assigned when the cluster state changes in a significant way. There are other factors, such as memory usage, that will affect whether or not persistent tasks are assigned to nodes but do not otherwise cause the cluster state to change. This setting defines how often assignment checks are performed in response to these factors. The default is `30 seconds`, with a minimum of `10 seconds` being required.
```

Comment:
```suggestion
cluster.persistent_tasks.allocation.recheck_interval | Time unit | The cluster manager automatically checks whether or not persistent tasks need to be assigned when the cluster state changes in a significant way. There are other factors, such as memory usage, that will affect whether or not persistent tasks are assigned to nodes but do not otherwise cause the cluster state to change. This setting defines how often assignment checks are performed in response to these factors. Default is `30 seconds`, with a minimum of `10 seconds` being required.
```